src/main/java/com/netgrif/application/engine/elastic/service/ElasticPetriNetService.java

```diff
@@ -0,0 +1,93 @@
+package com.netgrif.application.engine.elastic.service;
+
+import com.netgrif.application.engine.elastic.domain.ElasticPetriNet;
+import com.netgrif.application.engine.elastic.domain.ElasticPetriNetRepository;
+import com.netgrif.application.engine.elastic.service.executors.Executor;
+import com.netgrif.application.engine.elastic.service.interfaces.IElasticPetriNetService;
+import com.netgrif.application.engine.petrinet.domain.PetriNet;
+import com.netgrif.application.engine.petrinet.service.interfaces.IPetriNetService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.dao.InvalidDataAccessApiUsageException;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ElasticPetriNetService implements IElasticPetriNetService {
+    private static final Logger log = LoggerFactory.getLogger(ElasticPetriNetService.class);
```

Comment:
Please use `@Slf4j`. In java import `lombok.extern.slf4j.Slf4j` in groovy `groovy.util.logging.Slf4j`